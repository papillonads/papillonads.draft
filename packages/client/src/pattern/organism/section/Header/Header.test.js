import headerDefault from './Header.render'

describe('<Header />', () => {
  describe('Snaphot', () => {
    test('must match headerDefault', () => {
      expect(global.renderToJSON(headerDefault)).toMatchSnapshot()
    })
  })
})
