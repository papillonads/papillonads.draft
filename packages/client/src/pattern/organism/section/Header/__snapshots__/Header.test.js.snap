// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Snaphot must match defaultRender 1`] = `
<div
  className="header"
>
  <div
    className="header__navigationBar navigationBar"
  >
    <div
      className="navigationBar__maxWidth"
    >
      <div
        className="navigationBar__maxWidthLeft"
      >
        <div
          className="navigationBar__maxWidthLeftLogo logo"
        >
          <svg
            className="svgIconGroup"
            height="26"
            viewBox="0 0 528 96"
            width="143"
          >
            <g
              fillRule="evenodd"
            >
              <path
                d="M119.692 23.506v6.355l3.857.219-2.328 33.089c-.219 1.826-.583 2.337-1.965 2.556l-2.256.22v6.135h17.319v-6.355l-3.639-.219 1.456-26.296h.145l10.26 27.684h7.641l10.552-28.049h.218l1.237 24.397c.073 1.826-.218 2.264-1.601 2.483l-2.256.22v6.135h17.392v-6.355l-3.711-.219-2.256-33.235c-.219-1.534.51-2.191 1.892-2.41l2.183-.22v-6.135H157.75l-11.133 30.897h-.219l-10.697-30.897h-16.009zm92.488 13.221c-5.53-2.118-10.842-2.63-13.389-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.553 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zM202.14 59.955c0 2.922-2.11 5.624-6.113 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.567.365 4.513.73v17.604zm37.111-17.677c.8 0 1.383.073 1.82.22.509.219.654.584.654 1.752v4.821l8.223-.584V35.339c-1.965-.876-4.584-1.242-6.695-1.242-6.04 0-9.168 3.653-10.333 6.574h-.145v-5.624h-13.317v6.136l2.474.219c1.237.073 1.674.584 1.674 2.045V63.68c0 1.68-.218 2.191-1.746 2.41l-2.329.22v5.77h20.157v-6.063l-6.04-.292V49.071c0-3.871 2.11-6.793 5.603-6.793zm28.307 14.39c1.528-.22 2.91-.438 4.366-1.023l6.986 16.435h12.225v-6.063l-3.93-.219-6.622-14.316c3.493-2.849 5.458-6.501 5.967-10.007l3.275-.438v-5.99h-16.082v5.916l1.892.22c.946.219 1.528.438 1.528 1.387 0 3.36-5.166 6.501-9.605 7.086V20h-14.263v6.136l2.475.219c1.382.073 1.673.584 1.673 2.118v35.353c0 1.68-.436 2.045-1.673 2.264l-2.33.22v5.77h17.392v-6.063l-3.274-.219v-9.13zm25.032-14.536h4.948v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.277-.95 9.315-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.876 1.242-2.91 0-4.438-1.169-4.438-5.478V42.132h8.659l.364-7.085h-9.023v-8.4h-5.604c-1.528 4.383-3.492 8.254-8.732 9.203l-.873 6.282zM325.771 78.8c0 1.826-.364 2.118-1.746 2.337l-2.256.293v5.77h20.375v-6.063l-6.258-.365v-9.495c1.965 1.095 4.657 1.826 7.277 1.826 10.26 0 15.572-7.89 15.572-20.672 0-11.76-4.584-18.334-13.098-18.334-4.075 0-8.15 1.461-10.406 5.479h-.145v-4.529h-13.463v6.136l2.402.146c1.164.073 1.746.73 1.746 2.118V78.8zm10.188-31.263c0-2.337 1.892-5.697 6.112-5.697 5.312 0 6.404 5.843 6.404 11.833 0 6.647-1.674 11.833-7.131 11.833-1.82 0-3.93-.657-5.385-1.753V47.537zM376.854 20h-14.262v6.136l2.401.219c1.383.073 1.747.584 1.747 2.118V63.68c0 1.826-.291 2.118-1.674 2.337l-2.329.293v5.77h17.901v-6.063l-3.784-.219V20zm40.532 16.727c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm50.792-23.228c-5.53-2.118-10.842-2.63-13.39-2.63-13.17 0-19.574 8.254-19.574 21.11 0 9.934 3.784 17.823 14.554 17.823 4.657 0 7.932-1.973 9.678-4.675l.291.073.291 3.652h11.789v-5.843l-2.038-.147c-1.237-.073-1.6-.584-1.6-2.045V36.727zm-10.042 23.228c0 2.922-2.11 5.624-6.112 5.624-5.24 0-6.55-5.259-6.55-10.883 0-6.94 1.82-13.075 8.15-13.075 1.82 0 3.566.365 4.512.73v17.604zm16.227-17.823h4.949v17.896c0 6.209.582 13.075 11.934 13.075 3.784 0 7.276-.95 9.314-2.338l-1.674-6.939c-1.164.657-2.91 1.242-4.875 1.242-2.91 0-4.439-1.169-4.439-5.478V42.132h8.66l.363-7.085h-9.023v-8.4h-5.603c-1.528 4.383-3.493 8.254-8.732 9.203l-.874 6.282zm49.192-.657c.873.292 1.018.803 1.018 1.972v2.995l7.204-.585v-9.13c-3.201-1.753-7.567-2.63-11.57-2.63-8.732 0-15.208 4.164-15.208 11.98 0 7.888 5.53 9.568 10.77 10.956 4.074.95 6.912 1.68 6.912 4.82 0 3.142-2.255 4.237-5.675 4.237-1.456 0-2.984-.292-3.93-.657-.873-.292-1.091-.73-1.091-1.826V60.32l-7.277.584v9.715c2.838 1.534 8.44 2.484 12.88 2.484 9.387 0 15.354-4.456 15.354-12.345 0-7.596-5.749-9.276-11.061-10.883-3.857-1.169-6.986-1.753-6.986-5.04 0-2.557 2.183-3.798 5.094-3.798 1.164 0 2.474.073 3.566.438z"
                fill="#2D3C4D"
              />
              <path
                d="M48.018 0C21.498 0 0 21.492 0 48s21.497 48 48.018 48 48.019-21.492 48.019-48S74.539 0 48.018 0"
                fill="#EEA566"
              />
              <path
                d="M64.19 36.65a8.877 8.877 0 0 1 0 12.782L51.196 62.09c-3.623 3.533-9.497 3.533-13.12 0l19.554-19.05L44.64 30.38a8.874 8.874 0 0 1 0-12.78l19.55 19.05zM57.21 77.12H29.771c0-4.995 4.12-9.047 9.206-9.047H66.417c0 4.996-4.121 9.047-9.206 9.047"
                fill="#2D3C4D"
              />
            </g>
          </svg>
        </div>
        <div
          className="navigationBar__maxWidthLeftLinks links"
        >
          <div
            className="link link"
          >
            Help en info
          </div>
          <div
            className="link link"
          >
            Voorwaarden
          </div>
          <div
            className="link link"
          >
            Veilig handelen
          </div>
        </div>
      </div>
      <div
        className="_menu"
      >
        <div
          className="_menu__messages badge"
        >
          <div
            className="_menu__messagesIcon svgIcon"
          >
            <svg
              fill="grey"
              height={15}
              viewBox="0 0 1024 1024"
              width={15}
            >
              <path
                d="M480 128c-50.666 0-99.582 7.95-145.386 23.628-42.924 14.694-81.114 35.436-113.502 61.646-60.044 48.59-93.112 110.802-93.112 175.174 0 35.99 10.066 70.948 29.92 103.898 20.686 34.34 51.898 65.794 90.26 90.958 30.44 19.968 50.936 51.952 56.362 87.95 0.902 5.99 1.63 12.006 2.18 18.032 2.722-2.52 5.424-5.114 8.114-7.794 24.138-24.040 56.688-37.312 90.322-37.312 5.348 0 10.718 0.336 16.094 1.018 19.36 2.452 39.124 3.696 58.748 3.696 50.666 0 99.58-7.948 145.384-23.628 42.926-14.692 81.116-35.434 113.504-61.644 60.046-48.59 93.112-110.802 93.112-175.174s-33.066-126.582-93.112-175.174c-32.388-26.212-70.578-46.952-113.504-61.646-45.804-15.678-94.718-23.628-145.384-23.628zM480 0v0c265.096 0 480 173.914 480 388.448s-214.904 388.448-480 388.448c-25.458 0-50.446-1.62-74.834-4.71-103.106 102.694-222.172 121.108-341.166 123.814v-25.134c64.252-31.354 116-88.466 116-153.734 0-9.106-0.712-18.048-2.030-26.794-108.558-71.214-177.97-179.988-177.97-301.89 0-214.534 214.904-388.448 480-388.448zM996 870.686c0 55.942 36.314 104.898 92 131.772v21.542c-103.126-2.318-197.786-18.102-287.142-106.126-21.14 2.65-42.794 4.040-64.858 4.040-95.47 0-183.408-25.758-253.614-69.040 144.674-0.506 281.26-46.854 384.834-130.672 52.208-42.252 93.394-91.826 122.414-147.348 30.766-58.866 46.366-121.582 46.366-186.406 0-10.448-0.45-20.836-1.258-31.168 72.57 59.934 117.258 141.622 117.258 231.676 0 104.488-60.158 197.722-154.24 258.764-1.142 7.496-1.76 15.16-1.76 22.966z"
              />
            </svg>
          </div>
          <div
            className="_menu__messagesLabel label"
          >
            Berichten
          </div>
        </div>
        <div
          className="_menu__notifications badge"
        >
          <div
            className="_menu__notificationsIcon svgIcon"
          >
            <svg
              fill="grey"
              height={15}
              viewBox="0 0 1024 1024"
              width={15}
            >
              <path
                d="M1025.5 800c0-288-256-224-256-448 0-18.56-1.788-34.42-5.048-47.928-16.83-113.018-92.156-203.72-189.772-231.36 0.866-3.948 1.32-8.032 1.32-12.21 0-33.278-28.8-60.502-64-60.502s-64 27.224-64 60.5c0 4.18 0.456 8.264 1.32 12.21-109.47 30.998-190.914 141.298-193.254 273.442-0.040 1.92-0.066 3.864-0.066 5.846 0 224.002-256 160.002-256 448.002 0 76.226 170.59 139.996 398.97 156.080 21.524 40.404 64.056 67.92 113.030 67.92s91.508-27.516 113.030-67.92c228.38-16.084 398.97-79.854 398.97-156.080 0-0.228-0.026-0.456-0.028-0.682l1.528 0.682zM826.246 854.096c-54.23 14.47-118.158 24.876-186.768 30.648-5.704-65.418-60.582-116.744-127.478-116.744s-121.774 51.326-127.478 116.744c-68.608-5.772-132.538-16.178-186.768-30.648-74.63-19.914-110.31-42.19-123.368-54.096 13.058-11.906 48.738-34.182 123.368-54.096 86.772-23.152 198.372-35.904 314.246-35.904s227.474 12.752 314.246 35.904c74.63 19.914 110.31 42.19 123.368 54.096-13.058 11.906-48.738 34.182-123.368 54.096z"
              />
            </svg>
          </div>
          <div
            className="_menu__notificationsLabel label"
          >
            Meldingen
          </div>
        </div>
        <details
          className="details-reset dropdown"
        >
          <summary
            aria-haspopup="true"
            className="dropdown__summary"
          >
            <div
              className="dropdown__badge badge"
            >
              <div
                className="dropdown__svgIcon1 svgIcon"
              >
                <svg
                  fill="grey"
                  height={20}
                  viewBox="0 0 1024 1024"
                  width={20}
                >
                  <path
                    d="M576 706.612v-52.78c70.498-39.728 128-138.772 128-237.832 0-159.058 0-288-192-288s-192 128.942-192 288c0 99.060 57.502 198.104 128 237.832v52.78c-217.102 17.748-384 124.42-384 253.388h896c0-128.968-166.898-235.64-384-253.388z"
                  />
                </svg>
              </div>
              <div
                className="dropdown__label label"
              >
                t
              </div>
              <div
                className="dropdown__svgIcon2 svgIcon"
              >
                <svg
                  fill="grey"
                  height={15}
                  viewBox="0 0 1024 1024"
                  width={15}
                >
                  <path
                    d="M287.986 576c8.908 0 17.77 3.698 24.096 10.928l199.918 228.478 199.918-228.478c11.636-13.3 31.856-14.65 45.154-3.010 13.3 11.638 14.648 31.854 3.010 45.154l-224 256c-6.076 6.944-14.854 10.928-24.082 10.928s-18.006-3.984-24.082-10.928l-224-256c-11.638-13.3-10.292-33.516 3.010-45.154 6.070-5.312 13.582-7.918 21.058-7.918z"
                  />
                </svg>
              </div>
            </div>
          </summary>
          <ul
            className="dropdown-menu dropdown__list"
          >
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Advertenties
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Favorieten
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Zoekopdrachten
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Verkopers
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Profiel
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Mijn Ervaringen
              </div>
            </li>
            <li>
              <div
                className="dropdown-item dropdown__listItem label"
              >
                Uitloggen
              </div>
            </li>
          </ul>
        </details>
        <div
          className="_menu__placeAd badge"
        >
          <div
            className="_menu__placeAdIcon svgIcon"
          >
            <svg
              fill="white"
              height={15}
              viewBox="0 0 1024 1024"
              width={15}
            >
              <path
                d="M544 0l-96 96 96 96-224 256h-224l176 176-272 360.616v39.384h39.384l360.616-272 176 176v-224l256-224 96 96 96-96-480-480zM448 544l-64-64 224-224 64 64-224 224z"
              />
            </svg>
          </div>
          <div
            className="_menu__placeAdButton button"
          >
            Plaats Advertentie
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="header__searchBar searchBar"
  >
    <div
      className="searchBar__maxWidth"
    >
      <div
        className="searchBar__maxWidthSearchFieldsetStandard searchFieldsetStandard"
      >
        <input
          className="searchFieldsetStandard__input input"
          placeholder=""
        />
      </div>
      <div
        className="searchBar__maxWidthSearchFieldsetAdvanced searchFieldsetAdvanced"
      >
        <div
          className="css-1pcexqc-container searchFieldsetAdvanced__group"
          onKeyDown={[Function]}
        >
          <div
            className="css-bg1rzq-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="css-1hwfws3"
            >
              <div
                className="css-151xaom-placeholder"
              >
                Alle groepen...
              </div>
              <div
                className="css-1g6gooi"
              >
                <div
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                    }
                  }
                >
                  <input
                    aria-autocomplete="list"
                    autoCapitalize="none"
                    autoComplete="off"
                    autoCorrect="off"
                    disabled={false}
                    id="react-select-2-input"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    spellCheck="false"
                    style={
                      Object {
                        "background": 0,
                        "border": 0,
                        "boxSizing": "content-box",
                        "color": "inherit",
                        "fontSize": "inherit",
                        "label": "input",
                        "opacity": 1,
                        "outline": 0,
                        "padding": 0,
                        "width": "1px",
                      }
                    }
                    tabIndex="0"
                    type="text"
                    value=""
                  />
                  <div
                    style={
                      Object {
                        "height": 0,
                        "left": 0,
                        "overflow": "scroll",
                        "position": "absolute",
                        "top": 0,
                        "visibility": "hidden",
                        "whiteSpace": "pre",
                      }
                    }
                  >
                    
                  </div>
                </div>
              </div>
            </div>
            <div
              className="css-1wy0on6"
            >
              <span
                className="css-bgvzuu-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className="css-16pqwjk-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="css-19bqh2r"
                  focusable="false"
                  height={20}
                  viewBox="0 0 20 20"
                  width={20}
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
        <input
          className="searchFieldsetAdvanced__postcode input"
          placeholder="Postcode"
        />
        <div
          className="css-1pcexqc-container searchFieldsetAdvanced__distance"
          onKeyDown={[Function]}
        >
          <div
            className="css-bg1rzq-control"
            onMouseDown={[Function]}
            onTouchEnd={[Function]}
          >
            <div
              className="css-1hwfws3"
            >
              <div
                className="css-151xaom-placeholder"
              >
                Alle afstanden...
              </div>
              <div
                className="css-1g6gooi"
              >
                <div
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                    }
                  }
                >
                  <input
                    aria-autocomplete="list"
                    autoCapitalize="none"
                    autoComplete="off"
                    autoCorrect="off"
                    disabled={false}
                    id="react-select-3-input"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    spellCheck="false"
                    style={
                      Object {
                        "background": 0,
                        "border": 0,
                        "boxSizing": "content-box",
                        "color": "inherit",
                        "fontSize": "inherit",
                        "label": "input",
                        "opacity": 1,
                        "outline": 0,
                        "padding": 0,
                        "width": "1px",
                      }
                    }
                    tabIndex="0"
                    type="text"
                    value=""
                  />
                  <div
                    style={
                      Object {
                        "height": 0,
                        "left": 0,
                        "overflow": "scroll",
                        "position": "absolute",
                        "top": 0,
                        "visibility": "hidden",
                        "whiteSpace": "pre",
                      }
                    }
                  >
                    
                  </div>
                </div>
              </div>
            </div>
            <div
              className="css-1wy0on6"
            >
              <span
                className="css-bgvzuu-indicatorSeparator"
              />
              <div
                aria-hidden="true"
                className="css-16pqwjk-indicatorContainer"
                onMouseDown={[Function]}
                onTouchEnd={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="css-19bqh2r"
                  focusable="false"
                  height={20}
                  viewBox="0 0 20 20"
                  width={20}
                >
                  <path
                    d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                  />
                </svg>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="searchBar__maxWidthButton button"
      >
        Zoek
      </div>
    </div>
  </div>
</div>
`;
